targets/LinuxDeployDependencies.proj

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="ProcessJsonFiles" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="DeployDependencies.proj" />
+  <Import Project="$(NuGetConfigDir)/packageLoad.targets" />
+  <Import Project="../override.targets" />
+
+  <PropertyGroup>
+    <NuGetRuntimeIdentifier Condition="'$(NetCoreBuild)' == 'true'">ubuntu.14.04-x64</NuGetRuntimeIdentifier>
+    <NuGetRuntimeIdentifier Condition="'$(NetCoreBuild)' != 'true'">win7-x64</NuGetRuntimeIdentifier>
+  </PropertyGroup>
+
+  <ItemGroup>
+     <RuntimeProjectJsonFile Condition="'$(NetCoreBuild)' == 'true'" Include="$(MSBuildThisFileDirectory)\runtime.project.json"/>
+     <RuntimeProjectJsonFile Condition="'$(NetCoreBuild)' != 'true'" Include="$(MSBuildThisFileDirectory)\runtime.project.json"/>
+  </ItemGroup>
+
+  <ItemGroup>
+     <RuntimeProjectJsonLockFile Include="@(RuntimeProjectJsonFile->'%(Directory)%(Filename).lock.json')"/>
+     <IntermediateRuntimeProjectJsonFile Include="@(RuntimeProjectJsonFile->'$(IntermediateOutputPath)%(Filename)\project.json')" />
+     <IntermediateRuntimeProjectJsonLockFile Include="@(RuntimeProjectJsonFile->'$(IntermediateOutputPath)%(Filename)\project.lock.json')" />
+  </ItemGroup>
+
+  <Target Name="_CopyIntermediateProjectJson"
+          Inputs="@(RuntimeProjectJsonFile)"
+          Outputs="@(IntermediateRuntimeProjectJsonFile)"
+          BeforeTargets="RestoreRuntimePackagesDnu"
+  >
+    <Copy SourceFiles="@(RuntimeProjectJsonFile)"
+          DestinationFiles="@(IntermediateRuntimeProjectJsonFile)"
+          SkipUnchangedFiles="true"
+    />
+  </Target>     
+
+  <Target Name="_CopyIntermediateProjectLockJson"
+          BeforeTargets="RestoreRuntimePackagesDnu"
+          Inputs="@(RuntimeProjectJsonLockFile)"
+          Outputs="@(IntermediateRuntimeProjectJsonLockFile)"
+  >
+    <Copy SourceFiles="@(RuntimeProjectJsonLockFile)"
+          DestinationFiles="@(IntermediateRuntimeProjectJsonLockFile)"
+          SkipUnchangedFiles="true"
+          ContinueOnError="true"
+    />
+  </Target>     
+
+  <Target Name="_CopyProjectLockJson"
+          Inputs="@(IntermediateRuntimeProjectJsonLockFile)"
+          Outputs="@(RuntimeProjectJsonLockFile)"
+          AfterTargets="RestoreRuntimePackagesDnu"
+  >
+    <Copy SourceFiles="@(IntermediateRuntimeProjectJsonLockFile)"
+          DestinationFiles="@(RuntimeProjectJsonLockFile)"
+          SkipUnchangedFiles="true"
+    />
+    
+    <!-- It appears that NuGet doesn't re-write the project.lock.json file if the contents would be the same. This can mess up incremental builds
+        if you make a non-significant change to project.json.  To avoid that, update the timestamp of the lock file here. -->
+    <Touch Files="@(RuntimeProjectJsonLockFile)" />
```

Comment:
I'd rather avoid duplicating this stuff if possible--common `.targets` imported by two projects?